src/msvc_helper_main-win32.cc

```diff
@@ -113,12 +113,13 @@ int MSVCHelperMain(int argc, char** argv) {
     PushPathIntoEnvironment(env);
   }
 
-  char* command = GetCommandLineA();
-  command = strstr(command, " -- ");
-  if (!command) {
+  wchar_t* w_command = GetCommandLine();
+  std::string w_command_str = WideToUtf8(w_command);
```

Comment:
Wouldn't this be better if named "utf8_command_str" ? the prefix of "w_" made me think it was a std::wstring.